src/main/resources/application.yml

```diff
@@ -7,7 +7,7 @@ folio:
   schedule:
     acquisition:
       poolSize: 10
-      runOnlyIfModuleRegistered: true
+      runOnlyIfModuleRegistered: false
```

Comment:
Same, that's just for debugging purposes. This is used for inmemory scheduling and should be left as is for the commit. Quartz impl does not use this property

Reply:
Fixed.